### DISCUSSION TITLE
force redraw

### DISCUSSION THREAD

**Discussion by nschloe** [pyvista/pyvista#2648] [Q&A]
Sometimes when plotting and adjusting the camera, the first render would show nothing:

![sc1](https://user-images.githubusercontent.com/181628/130827928-eb2e083c-58fe-465d-8fd7-21992df178c7.png)

Only after I update the camera by clicking and dragging anywhere with the mouse, I see the object:

![sc2](https://user-images.githubusercontent.com/181628/130827929-0004e156-9bcd-4a22-9b7d-4d066faa22f2.png)

Is there any way to force a redraw in software?

**Comment by akaszynski** [comment]
This is likely due to the camera clipping range not being reset.  See [pyvista.BasePlotter.reset_camera_clipping_range](https://docs.pyvista.org/plotting/plotting.html?highlight=camera%20clipping#pyvista.BasePlotter.reset_camera_clipping_range).

**Comment by nschloe** [comment]
Indeed, thanks!
